cfme/tests/openstack/test_host_power_control.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon
+from cfme.infrastructure.host import Host
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'], scope='module')
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_host_power_off(provider):
+    nodes_bool = provider.load_all_provider_nodes()
+    if not nodes_bool:
+        assert "Missing nodes in provider's details"
+    my_quads = list(Quadicon.all())
+    quad = my_quads[0]
+    host = Host(name=quad.name)
+    if host.get_power_state() == 'on':
+        host.power_off()
+        host.refresh()
+    elif host.get_power_state() == 'off':
+        pytest.skip("Node is already off")
+    else:
+        pytest.fail("Power State is unknown")
+
+    host.wait_for_host_state_change('off', timeout=300)
+    assert host.get_power_state() == 'off'
```

Comment:
**REQUIRED:** This isn't strictly necessary. As the ```wait_for``` will raise a ```TimedOutError``` if the state doesn't change. This means the assertion will only ever match a success, at which point it is pointless.